lib/nsxv.py

```diff
@@ -209,6 +209,11 @@ def __str__(self):
     source_addr = []
     destination_addr = []
 
+    source_ipv4_address = False
+    source_ipv6_address = False
+    dest_ipv4_address = False
+    dest_ipv6_address = False
```

Comment:
Another option would be if there is a way to guarantee that naming always returns a sorted list of address. You could compare the first and last values and if they are the same AF family then it only contains one family. This would definitely simplify the process. I took a look at naming and it does appear to sort the returned result but haven't tested if it actually makes it way back through policyreader, then policy then generators to confirm.